backend/lp_generator.py

```
@@ -132,8 +130,8 @@ def save_to_file(html_content, file_name):
 ## APIを使って画像生成するコード
 @ray.remote
 def generate_image_by_imagen3(prompt, file_name, aspect_ratio=None):
-    # APIクライアントの初期化
-    client = genai_img.Client(api_key=os.environ.get("GOOGLE_IMAGEN_API_KEY"))
+    from PIL import Image, ImageDraw, ImageFont
```

Ruff (0.12.2)

Redefinition of unused `Image` from line 7

Remove definition: `Image`

(F811)